### ISSUE TITLE
Markdown Previewer Test #4

### ISSUE THREAD

**Issue by chrisbchrist** [freeCodeCamp/testable-projects-fcc#677]
#### Issue Description
My pen is failing test #4 but I don't believe it should be.  The editor loads default example code which is rendered to HTML in the preview, and any additional markdown is rendered correctly and apart from styling it functions identically to the example pen.  I get this error message:

The markdown in #editor is not being interpreted correctly and/or rendered into #preview : expected '<p>testing and…</p>\n' to equal '<p>testing and...</p>\n'
AssertionError: The markdown in #editor is not being interpreted correctly and/or rendered into #preview : expected '<p>testing and…</p>\n' to equal '<p>testing and...</p>\n'
    at s.u (https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js:262:1134)
    at s.e.(anonymous function) [as equal] (https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js:325:126)
    at Function.n.strictEqual (https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js:224:568)
    at r.<anonymous> (https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js:154:286395)
    at c (https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js:159:31608)
    at o.f.run (https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js:159:31544)
    at m.runTest (https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js:159:37114)
    at https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js:159:37976
    at s (https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js:159:36426)
    at https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js:159:36495

#### Browser Information

* Browser Name, Version:  Latest Chrome/edge
* Operating System: Windows 10
* Mobile, Desktop, or Tablet: Desktop

#### Your Code / Link to Your Pen

```
 https://codepen.io/extrachrisb/pen/BqgoZx
 
 
```

**Comment by ValeraS** [comment]
@chrisbchrist don't use a `smartypants` option, with it marked replaces `...` with `…` and test can't handle it.

But without the option test # 6 fails. It happens because in the renderer Marked passes `'` as `&#39;`.

To pass the test you could remove `'` from headings in your initial contents.

**Comment by chrisbchrist** [comment]
Thanks so much Valeriy, the apostrophes were the culprit!  Nobody ever suspects the contractions!  Issue resolved.

**Comment by Bus42** [comment]
I have tried renaming, restructuring, rewriting, refactoring, regurgitating, recycling, redundant repetition, and every possible thing I can think of. I've removed apostrophes, changed single to double quotations and vice versa, even writing code that shouldn't be written just to pass, and nothing makes #4 pass. The test is faulty plain and simple. There are way too many people having the same problem as me for it to be anything but the way the test is structured. It took approximately 5 minutes to make a markdown previewer, but for the last week and a half, I've been trying to get the thing to pass one @#$!*&% test!!
